src/components/CreateRules/UploadRules.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-card flat id="upload-rules">
+
+    <!-- Summary Section -->
+    <div id="upload-rules-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="upload-rules-summary-header" >
+        <v-icon color="#38598A">mdi-format-list-text</v-icon>
```

Comment:
I suggest we add named colours to our Vuetify themes declaration, instead of using hex colours throughout the code.

Reply:
Ah yes, i did add them to the theme, perhaps forgot to update them here in the placeholder components.

Reply:
You didn't add this colour to the Vuetify theme ... pick a nice semantic name for it and then you can use it in Vuetify elements/components.

Reply:
oh this was a carry over from the pre existing icons in the summary headers! Ok, will update, for ALL of them 👍

Reply:
Updated in next commit 👍